packages/calcite-components/src/tests/commonTests/cancelableResource.ts

```diff
@@ -0,0 +1,35 @@
+import { mount } from "@arcgis/lumina-compiler/testing";
```

Comment:
Can you move this into a different location within `commonTests`? Browser mode tests are not compatible with Puppeteer ones and we want to discourage mixing.